examples-dot-net/README.md

```diff
@@ -0,0 +1,1128 @@
+# Microsoft Azure Stack HCI management client library for .NET
+
+Microsoft Azure VMware Solution is a VMware validated solution with ongoing validation and testing of enhancements and upgrades. Microsoft manages and maintains the private cloud infrastructure and software. It allows you to focus on developing and running workloads in your private clouds to deliver business value.
+
+This library supports managing Microsoft Azure Stack HCI resources.
+
+This library follows the [new Azure SDK guidelines](https://azure.github.io/azure-sdk/general_introduction.html), and provides many core capabilities:
+
+    - Support MSAL.NET, Azure.Identity is out of box for supporting MSAL.NET.
+    - Support [OpenTelemetry](https://opentelemetry.io/) for distributed tracing.
+    - HTTP pipeline with custom policies.
+    - Better error-handling.
+    - Support uniform telemetry across all languages.
+
+## Getting started
+
+### Install the package
+
+Install the Azure Stack HCI management library for .NET with [NuGet](https://www.nuget.org/):
+
+```dotnetcli
+dotnet add package Azure.ResourceManager.Hci
+```
+
+### Prerequisites
+
+* You must have an [Microsoft Azure subscription](https://azure.microsoft.com/free/dotnet/).
+
+### Authenticate the Client
+
+To create an authenticated client and start interacting with Microsoft Azure resources, see the [quickstart guide here](https://github.com/Azure/azure-sdk-for-net/blob/main/doc/dev/mgmt_quickstart.md).
+
+## Key concepts
+
+Key concepts of the Microsoft Azure SDK for .NET can be found [here](https://azure.github.io/azure-sdk/dotnet_introduction.html).
+
+## Documentation
+
+Documentation is available to help you learn how to use this package:
+
+- [Quickstart](https://github.com/Azure/azure-sdk-for-net/blob/main/doc/dev/mgmt_quickstart.md).
+- [API References](https://docs.microsoft.com/dotnet/api/?view=azure-dotnet).
+- [Authentication](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/README.md).
+
+## Examples
+
+### Hci Cluster Collection
+
+#### Get list of all clusters in a given resource group()
+1. Get the Azure token
+
+```C# Snippet: 
+            TokenCredential cred = new DefaultAzureCredential();
+            ArmClient client = new ArmClient(cred);
+```
+2. Update the parameters given below
+```C# Snippet:
+            string subscription = "00000000-0000-0000-0000-000000000000"; # Replace with your subscription ID
+            string resourceGroupName = "hcicluster-rg"; # Replace with your resource group name
+```
+3. Get the collection of the Hci Cluster Resource
+```C# Snippet:
+            HciClusterCollection collection = resourceGroupResource.GetHciClusters();
+```
+4. Get Resource group resource
```

Comment:
We don't need this

Reply:
meaning the entire get list of all resource group function is not needed?